roles/galaxyprojectdotorg.galaxy/tasks/main.yml

```diff
@@ -2,7 +2,7 @@
 # main tasks file for galaxyprojectdotorg.galaxy
 
 # Deploy a Galaxy server
-#   https://wiki.galaxyproject.org/
+#   https://galaxyproject.org/
```

Comment:
We'll need to update this over in galaxyproject/ansible-galaxy, but I'll do that later.